packages/docs/cookbook/composing-stores.md

```diff
@@ -40,34 +40,12 @@ const useY = defineStore('y', () => {
 })
 ```
 
-## Nested stores
+## Using a store within another store
 
-Note that if one store uses another store, **there is no need to create a new store in a separate file**, you can directly import it. Think of it as nesting.
+Note that if one store uses another store, you can directly import and call the `use${storeName}Store` method, then access methods on the returned store object, just like you would access them from within a Vue component.
 
-You can call `useOtherStore()` at the top of any getter or action:
+You can call `useOtherStore()` inside any getter or action:
 
-```js
-import { useUserStore } from './user'
-
-export const cartStore = defineStore('cart', {
```

Comment:
let's keep this example but transform it into the setup store syntax:

```ts
defineStore('cart', () => {
  // ...
})
```

Reply:
Hi @posva, thanks for feedback. I've made the other changes but I'm confused by this one, and it might just be my lack of familiarity with the code.

1) I'm not sure how to write this example with the syntax you're asking. I've only found out now that `defineStore` accepts a function rather than an Object as its second argument- Pinia docs mostly demonstrate using an Object, and my org only uses this approach. [It seems like this confusion is shared](https://www.reddit.com/r/vuejs/comments/snpo86/comment/hw6sv1j/?utm_source=reddit&utm_medium=web2x&context=3), and this document is perhaps the only part of the docs that demonstrates passing a `storeSetup` function, in its infinite loop example above. There's probably already a push to document this more, but maybe we could start with other parts of the docs like [Defining A Store](https://pinia.vuejs.org/core-concepts/#defining-a-store)? I can make a PR if that's helpful
2) I removed this example because it seemed redundant to demonstrate the nested stores twice- we're already demonstrating it in the next two sections. Are you suggesting we keep it just to demonstrate the storeSetup syntax?

Reply:
update: [I've made a PR](https://github.com/vuejs/pinia/pull/1475) to document the setup store syntax elsewhere in the docs

Reply:
Cool, thanks!

I will add the example after merging